packages/server/src/sdk/app/rows/search.ts

```diff
@@ -81,6 +81,10 @@ export async function search(
       options.query = {}
     }
 
+    if (context) {
+      options.query = await enrichSearchContext(options.query, context)
+    }
```

Comment:
This has been moved from line 140 to here. This has to happen before `searchInputMapping` because `searchInputMapping` is what corrects metadata user IDs to global ones.